samples/internat/internat.cpp

```diff
@@ -327,10 +326,17 @@ MyFrame::MyFrame(wxLocale& locale)
     macro_menu->Append(INTERNAT_MACRO_8, wxGETTEXT_IN_CONTEXT_PLURAL("context_2", "sing", "plur", 1));
     macro_menu->Append(INTERNAT_MACRO_9, wxGETTEXT_IN_CONTEXT_PLURAL("context_2", "sing", "plur", 2));
 
+    wxMenu *help_menu = new wxMenu;
```

Comment:
I typically try to keep the style locally consistent first, before making it globally consistent, so I'd rather keep it like this.

I'm still waiting for clang-format to become good enough to run it on our code base to harmonize the style of it once and for all...

Reply:
Thanks, I'll stick to local consistency then. I do like the incremental changes: as a reader they can give me small hints which code is newer. The inconsistency bothers me less in such cases. Also it means a few drops of blame noise less with a global format replacement ;).

> I'm still waiting for clang-format to become good enough to run it on our code base to harmonize the style of it once and for all...

I haven't used it, but I thought it was 'just' a matter of writing a good enough formatting file. Or it's already known to not be adequate for wx?

Reply:
> > I'm still waiting for clang-format to become good enough to run it on our code base to harmonize the style of it once and for all...
> 
> I haven't used it, but I thought it was 'just' a matter of writing a good enough formatting file. Or it's already known to not be adequate for wx?

Last time I checked, it didn't support quite the style we use, so running it would result in an even greater amount of changes that it normally does. Maybe we should still do it though, it's true that fixing style all the time is tiresome.